src/lua/LuaSectorView.cpp

```diff
@@ -262,6 +271,44 @@ static int l_sectorview_get_factions(lua_State *l)
 	return 1;
 }
 
+static int l_sectorview_set_rotate_mode(lua_State *l)
```

Comment:
May I recommend you begin porting this interface to LuaMetaType? I'd love to see what bugs you shake out of the tree (and what features you'd like to see).

Reply:
Done!